src/src/AdminBundle/Controller/EventController.php

```diff
@@ -12,12 +13,16 @@
 class EventController extends Controller
 {
     /**
-     * @Route("/event/add", name="event_add")
+     * @Route("/event_type/edit/{id}/event/add", name="event_add")
```

Comment:
Toto bude asi chyba vsade kde pouzivame `{id}` ale len teraz som si to uvedomil. Kedze `{id}` nam matchne hocico, aj string musime pouzit https://symfony.com/doc/3.4/routing/requirements.html a povedat mu, ze ocakavame cislo.

Reply:
otestuj ako sa to sprava ked tam napises volovinu a potom vytvor prosim issue, kde tento problem spomenies nech na to nezabudneme.